crates/sui-core/src/test_utils.rs

```diff
@@ -150,11 +150,13 @@ async fn init_genesis(
         .into_iter()
         .cloned()
         .collect();
+    let (std_move_pkg, sui_move_pkg) = sui_framework::make_std_sui_move_pkgs();
     let pkg = Object::new_package(
         modules,
         OBJECT_START_VERSION,
         TransactionDigest::genesis(),
         ProtocolConfig::get_for_max_version().max_move_package_size(),
+        &[std_move_pkg, sui_move_pkg],
```

Comment:
Along with the change I mentioned to this function (not required or a big deal here, but this should be the preferred pattern):

```suggestion
        [&std_move_pkg, &sui_move_pkg],
```